### PR TITLE
#15559. added new fields for layers endpoints

### PR DESCRIPTION
**Pull request by kshmidman**

### CODE REVIEW

File: src/main/java/io/kontur/disasterninja/service/layers/LayersApiService.java [hunk 1]
```diff
@@ -114,14 +115,17 @@ private Layer convertToLayer(Collection collection, boolean includeSource, UUID
                 .group(collection.getGroup() != null ? collection.getGroup().getName() : null)
                 .legend(collection.getStyleRule() != null ?
                         JsonUtil.readObjectNode(collection.getStyleRule(), Legend.class) : null)
+                .style(collection.getStyle() != null ?
+                        JsonUtil.readObjectNode(collection.getStyle(), Style.class) : null)
```
Comment (i1shilyaev) [inline src/main/java/io/kontur/disasterninja/service/layers/LayersApiService.java:119]: Was it a requirement to parse JSON on BE side instead of FE?

**Review by i1shilyaev — APPROVED** [review]

**Comment by i1shilyaev** [comment]
Please, do not merge this branch into main, until there is clarity with tile proxying functionality (whether we should revert it or not)

File: src/main/java/io/kontur/disasterninja/service/layers/LayersApiService.java (hunk 1 above)
Comment (kshmidman) [reply src/main/java/io/kontur/disasterninja/service/layers/LayersApiService.java:119]: I don't know how it was agreed, but for legend it was done before. So I did the same way.

Comment (i1shilyaev) [reply src/main/java/io/kontur/disasterninja/service/layers/LayersApiService.java:119]: Then probably you should ask FE if it's okay for them

**Review by i1shilyaev — DISMISSED** [review]

**Review by i1shilyaev — APPROVED** [review]

**Pull request merged**